spec/lib/efile/nj/nj1040_spec.rb

```diff
@@ -104,6 +104,19 @@ def over_65_birth_year
     end
   end
 
+  describe 'line 13 - total exemptions' do
+    let(:intake) { create(:state_file_nj_intake) }
+    it 'sets line 13 to the sum of lines 6-12' do
```

Comment:
[pebble] might suggest that it should say the sum of lines 6-8, given that's what has been coded. Other lines are not being included in the sum nor tested yet